### DISCUSSION TITLE
When to start a project of my own?

### DISCUSSION THREAD

**Discussion by naveenburathi** [smartcontractkit/full-blockchain-solidity-course-py#481] [General]
Hlo everyone there,
I m currently in prefinal year of an engineering college. I want to prepare a project using blockchain. Currently I am on 5th lesson of this course. 
So I am having a doubt when to to start creating a own project. Should I wait patiently and finish the course first. Or should I start thinking and creating of a project side by side.
Your experience might help me a lot.
Thank You🙏

**Comment by cromewar** [comment]
Hello, it depends on how complex your project would be, if you were planning to use front end, take lesson 13 as reference, lesson 12 if you want to do NFTs. I think you could just start, even if you don't know what you are doing, you will learn a lot with your project and meanwhile you take the course.
